sway-core/src/asm_generation/fuel/allocated_abstract_instruction_set.rs

```diff
@@ -419,6 +419,13 @@ impl AllocatedAbstractInstructionSet {
                 }
             }
 
+            Either::Left(AllocatedOpcode::CFEI(ref op))
```

Comment:
Please add a comment here on why we're doing this. Something like "cfei 0 and cfsi 0 are omitted from actual asm emission hence we shouldn't count them for offsets"